### ISSUE TITLE
mat-form-field

### ISSUE THREAD

**Issue by euberdeveloper** [julianobrasil/multipicker#6]
The component is good, but when I try to put it inside a <mat-form-field-control> it throws an error wich says: mat-form-field must contain a MatFormFieldControl. Is there a support for mat-form-field?

**Comment by julianobrasil** [comment]
No, there isn't. Inside the component, I use `mat-form-field`. But I if you are trying to use the `appearance` `Input()`, I can expose it.

**Comment by euberdeveloper** [comment]
I need a mat-form-field wrapper because I can show things like error required. It would be beautiful to make your component without mat-form-field and to implement MatFormFieldControl in it. It would be like a native material component.

For now I made my component wich closes like yours the picker when year is selected.

In addition, it would be beautiful if there were the moment and moment-adapter dependencies in the lib package, so that when installed they are installed too.

**Comment by julianobrasil** [comment]
Yeah... it'd be definitely a nice improvement. I'll try to spare some time to do it in a few weeks.
